common/errcode.h

```diff
@@ -9,4 +9,21 @@ typedef s32 errcode_t;
 
 #define PRIerrcode PRId32
 
+// HSM errors code
+//FIXME(vincenzopalazzo) Make the correct calculation about error value
+#define HSM_ERROR_IS_ENCRYPT 20
+#define HSM_NOT_FOUND 21
+#define HSM_GENERIC_ERROR 22
+#define HSM_WRONG_PASSWD 23
+#define HSM_SHORT_PASSWD 24
+#define HSM_LONG_PASSWD 25
+#define HSM_DERIVATION_KEY 26
```

Comment:
I agree, I add this verbosity here because I was thinking that in case to manage a UI from a bash script like the raspblitz case we can help to define better error messages for the user.

But maybe this is too much for lightning-cli?